### PR TITLE
fix: landing page button layout

### PR DESCRIPTION
**Pull request by NickSaglioni**
- Fixing landing page button layout;
- Adding style for cards;
- Adding style for datatable;
- Minor adjustments in the primevue overrides.

### CODE REVIEW

File: style-sheets/primevue-components-overrides.scss [hunk 1]
```diff
@@ -271,5 +295,57 @@ label {
...
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-title {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    margin-bottom: 0;
+}
 
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-subtitle {
+    color: var(--light-theme-text-text-secondary, #606062);
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:320]: let's try to find if this color is in the design token folder

File: style-sheets/primevue-components-overrides.scss [hunk 2]
```diff
@@ -271,5 +295,57 @@ label {
...
+    letter-spacing: 0.16px;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
+    padding: 0;
+}
 /* ------------------------------------ primevue table --------------------------------- */
+.#{variables.$primevue-prefix}-datatable .#{variables.$primevue-prefix}-datatable-thead > tr > th {
+    /* Light Theme/Layer accent/$layer-accent-hover-01 */
+    background: #d2d2d4;
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:332]: let's use the design token for the color

File: style-sheets/primevue-components-overrides.scss [hunk 3]
```diff
@@ -271,5 +295,57 @@ label {
...
+    padding: 0 1rem;
+    font-size: 14px;
+    font-weight: 700;
+    letter-spacing: 0.1599999964237213px;
+    text-align: left;
+    border: none;
+}
+.#{variables.$primevue-prefix}-datatable .#{variables.$primevue-prefix}-datatable-tbody > tr > td {
+    text-align: left;
+    border: 1px solid #dee2e6;
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:343]: check if this color exists in the design tokens

File: style-sheets/primevue-components-overrides.scss (hunk 1 above)
Comment (NickSaglioni) [reply style-sheets/primevue-components-overrides.scss]: Thanks Catherine, for the comments. I have updated this.

File: style-sheets/primevue-components-overrides.scss (hunk 2 above)
Comment (NickSaglioni) [reply style-sheets/primevue-components-overrides.scss]: Thanks, this was updated.

File: style-sheets/primevue-components-overrides.scss (hunk 3 above)
Comment (NickSaglioni) [reply style-sheets/primevue-components-overrides.scss]: Thanks, I have updated it.

File: style-sheets/primevue-components-overrides.scss [hunk 4]
```diff
@@ -271,5 +296,60 @@ label {
 }
 
 /* ------------------------------------ primevue card--------------------------------- */
+.#{variables.$primevue-prefix}-card {
+    box-shadow: none;
+    border: 1px solid colors.$gray-30;
```
Comment (MCatherine1994) [inline style-sheets/primevue-components-overrides.scss:301]: let's check this color as well

File: style-sheets/primevue-components-overrides.scss [hunk 5]
```diff
@@ -5,19 +5,29 @@
...
 
 label {
-    font-weight: 400;
-    font-size: 14px;
-    color: map.get(lightTheme.$light-theme, 'text-primary');
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.32px;
+
+    color: colors.$gray-70;
```
[the remark this answers is not in the document]
Comment (NickSaglioni) [reply style-sheets/primevue-components-overrides.scss]: Thanks for spotting it, I could find a color for light-theme.

File: style-sheets/primevue-components-overrides.scss (hunk 4 above)
Comment (NickSaglioni) [reply style-sheets/primevue-components-overrides.scss]: Thanks for spotting it, I could find a color for light-theme.

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**